lib/rcedit.js

```diff
@@ -26,6 +27,13 @@ module.exports = function (exe, options, callback) {
     }
   })
 
+  noPrefixSettings.forEach(function (name) {
+    if (options[name] != null) {
```

Comment:
Other similar blocks accessing `options` in this file are using `!=` and it seems good to guard against both `undefined` and `null`.